platform/core/src/services/DisplaySetService/DisplaySetService.ts

```diff
@@ -62,11 +58,26 @@ export default class DisplaySetService extends PubSubService {
       // become important to do this in an efficient manner for large
       // numbers of display sets.
       if (!activeDisplaySets.includes(displaySet)) {
+        this.activeChanged = true;
         activeDisplaySets.push(displaySet);
       }
     });
   }
 
+  /**
+   * Adds new display sets
+   */
+  public addDisplaySets(...displaySets): string[] {
+    this._addDisplaySetsToCache(displaySets);
+    this._addActiveDisplaySets(displaySets);
+    this.activeChanged = false;
```

Comment:
why we are setting to `false` again here, need a comment. The whole function requires doc to say how is it different than `makeDisplaySet`